spec/requests/api/v1/merchants_requests_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe "Merchants API" do
+  it "returns all merchants" do
+    create_list(:merchant, 4)
+
+    get '/api/v1/merchants'
+
+    merchants = JSON.parse(response.body)
+    merchant = merchants.first
+
+    expect(response).to be_success
+    expect(merchants.count).to eq(4)
+    expect(merchant).to be_a(Hash)
+    expect(merchant).to have_key("id")
+    expect(merchant).to have_key("name")
+    expect(merchant).to have_key("created_at")
+    expect(merchant["id"]).to be_a(Integer)
+    expect(merchant["name"]).to be_a(String)
+  end
+
+  it "returns a specific merchant" do
+    merchants = create_list(:merchant, 4)
+    first_merchant = merchants.first
+    last_merchant = merchants.last
+
+    get '/api/v1/merchants/1'
+
+    merchant_1 = JSON.parse(response.body)
+
+    expect(response).to be_success
+    expect(merchant_1).to be_a(Hash)
+    expect(merchant_1).to have_key("id")
+    expect(merchant_1).to have_key("name")
+    expect(merchant_1).to have_key("created_at")
+    expect(merchant_1).to have_key("updated_at")
+    expect(merchant_1["id"]).to be_a(Integer)
+    expect(merchant_1["id"]).to eq(first_merchant["id"])
+    expect(merchant_1["name"]).to be_a(String)
+    expect(merchant_1["name"]).to eq(first_merchant["name"])
+
+    get '/api/v1/merchants/4'
+
```

Comment:
This seems like a very long test, but I like that it is testing the show page for another merchant. I'll merge anyways and we can talk about this tomorrow. I have a hunch that I will come to the same solution as I start writing my tests...